src/main/java/game/baseball/ball/Balls.java

```diff
@@ -0,0 +1,16 @@
+package game.baseball.ball;
+
+import java.util.List;
+
+public class Balls {
+
+    private final List<Integer> balls;
+
+    public Balls(List<Integer> balls) {
+        this.balls = balls;
+    }
+
+    public List<Integer> getBalls() {
+        return balls;
```

Comment:
일급 컬렉션을 고려해보시고 구현해주신거 같네요.
객체 상태는 외부에 노출이 될때, 상태를 변경하지 않도록 하는것이 중요합니다.
그렇기 떄문에 불변인 상태를 만드는 것이 좋은데요.
```suggestion
    return Collections.unmodifiableList(balls);
```
이런식으로 작성한다면 가능할거 같아요.

Reply:
외부에 객체가 노출 될 때 변경이 되면 안되는 값이니 최대한 보장 해주는게 좋군요 
Unmodifiable Collection에 대해서 몰랐는데 이번에 학습할 수 있었고
다른 방법으로 방어적 복사를 이용해서 
`new ArrayList<>(balls)` 이런 식으로 반환할 수도 있다는 것도 알 수 있었습니다 !